sharness/t0060-migration-stress.sh

```diff
@@ -4,6 +4,15 @@ test_description="Test migration 2 to 3 with lots of objects"
 
 . lib/test-lib.sh
 
+# setup vars for tests
+DEPTH=6
+PINTOTAL=2000
+if [ ! -z "$CI" ]; then
```

Comment:
Maybe we could try piping `ipfs add` into `tee`, so that its output still goes to stdout.
We will not detect if `ipfs add` fails, but that's probably ok.